tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -46,7 +46,7 @@ Create with anonymous volume
 
 Create with named volume
     ${disk-size}=  Run  docker %{VCH-PARAMS} logs $(docker %{VCH-PARAMS} start $(docker %{VCH-PARAMS} create -v test-named-vol:/testdir ${busybox} /bin/df -Ph) && sleep 10) | grep by-label | awk '{print $2}'
-    Should Be Equal As Strings  ${disk-size}  975.9M
+    Should Contain  ${disk-size}  975.9M
```

Comment:
Because we mount the vmdk rootfs and then bind mount its datadir, so when running the above command, two disks show up and the `975.9M` shows up twice so it no longer equals to the string with just one `975.9M`. That's why I had to change it to contain rather than equal to an exact string.